README.md

```diff
@@ -75,251 +60,76 @@ fn draw(frame: &mut Frame) {
-
-fn handle_events() -> std::io::Result<bool> {
-    match event::read()? {
-        Event::Key(key) if key.kind == KeyEventKind::Press => match key.code {
-            KeyCode::Char('q') => return Ok(true),
-            // handle other key events
-            _ => {}
-        },
-        // handle other events
-        _ => {}
-    }
-    Ok(false)
-}
-```
+You might want to checkout [Cursive](https://github.com/gyscos/Cursive) or
+[iocraft](https://github.com/ccbrown/iocraft/) for an alternative solutions
+to build text user interfaces in Rust.
 
-## Layout
+## Contributing
 
-The library comes with a basic yet useful layout management object called [`Layout`] which
-allows you to split the available space into multiple areas and then render widgets in each
-area. This lets you describe a responsive terminal UI by nesting layouts. See the [Layout]
-section of the [Ratatui Website] for more info.
+[![Discord Badge]][Discord Server] [![Matrix Badge]][Matrix]
+[![Forum Badge]][Forum]<br>
 
-```rust
-use ratatui::{
-    layout::{Constraint, Layout},
-    widgets::Block,
-    Frame,
-};
+Feel free to join our [Discord server](https://discord.gg/pMCEU9hNEj) for discussions and questions! There is also a [Matrix](https://matrix.org/) bridge available at
+[#ratatui:matrix.org](https://matrix.to/#/#ratatui:matrix.org). We have also recently launched the [Ratatui Forum][Forum].
 
-fn draw(frame: &mut Frame) {
-    use Constraint::{Fill, Length, Min};
+We rely on GitHub for [bugs][Report a bug] and [feature requests][Request a Feature].
 
-    let vertical = Layout::vertical([Length(1), Min(0), Length(1)]);
-    let [title_area, main_area, status_area] = vertical.areas(frame.area());
-    let horizontal = Layout::horizontal([Fill(1); 2]);
-    let [left_area, right_area] = horizontal.areas(main_area);
+Please make sure you read the [contributing](./CONTRIBUTING.md) guidelines before [creating a pull request][Create a Pull Request].
 
-    frame.render_widget(Block::bordered().title("Title Bar"), title_area);
-    frame.render_widget(Block::bordered().title("Status Bar"), status_area);
-    frame.render_widget(Block::bordered().title("Left"), left_area);
-    frame.render_widget(Block::bordered().title("Right"), right_area);
-}
-```
+## Acknowledgements
 
-Running this example produces the following output:
+- None of this could be possible without [Florian Dehau] who originally created [tui-rs] which inspired many Rust TUIs.
 
-```text
-Title Bar───────────────────────────────────
-┌Left────────────────┐┌Right───────────────┐
-│                    ││                    │
-└────────────────────┘└────────────────────┘
-Status Bar──────────────────────────────────
-```
+- Special thanks to [Pavel Fomchenkov] for his work in designing an awesome logo for the Ratatui project and organization.
 
-## Text and styling
+- A now unknown person in our community for coming up with the name "Ratatui".
```

Comment:
Nope, I don't remember who it was. Sayan just posted the suggestion on the naming poll if I remember correctly. (feel free to refresh my memory @sayanarijit)

I did some Discord archeology and [the first mention](https://discord.com/channels/1070692720437383208/1070692720437383211/1072962023442362409) of Ratatui is by @mrjackwills 

If my memory is not deceiving me, someone suggested that either in a meeting or sometime between the [1st naming poll](https://strawpoll.com/polls/BDyNEj7PwZR) and the second (there was two, because we couldn't decide).

And the original [google spreadsheet](https://docs.google.com/spreadsheets/d/1_9C9mYKFhN6ssW0Bg9aWjRsrdvKeik7WHS1j1W5e0bw/edit?usp=sharing) is now deleted - but it didn't have any credit to the people submitting names anyways:  

![image](https://github.com/user-attachments/assets/aee3ff29-f7e0-4194-a8a3-32529548db17)

Reply:
I didn't come up with the name, I think someone suggested it in the Discord, but I can't remember more than that

Reply:
@sayanarijit do you recall who came up with the name Ratatui?

Reply:
I will remove that entry until we figure this out :)